integration/spark/src/main/scala/org/apache/carbondata/spark/util/GlobalDictionaryUtil.scala

```diff
@@ -650,6 +650,51 @@ object GlobalDictionaryUtil extends Logging {
   }
 
   /**
+   * get file headers from fact file
+   *
+   * @param carbonLoadModel
+   * @return headers
+   */
+  private def getHeaderFormFactFile(carbonLoadModel: CarbonLoadModel): Array[String] = {
+    var headers: Array[String] = null
+    var factFile: String = null
+    val fileType = FileFactory.getFileType(carbonLoadModel.getFactFilePath)
+    val filePath = FileFactory.getCarbonFile(carbonLoadModel.getFactFilePath, fileType)
+    if (filePath.isDirectory) {
+      val listFiles = filePath.getParentFile.listFiles()
```

Comment:
Because we have inovked  FileUtils.getPaths to convert user's input to file list in LoadTable, so these codes are unnecessary.